Train.json

```diff
@@ -1383,6 +1383,21 @@
 			"cost": 380000,
 			"operationCosts": 120
 		},
+		{
+			"id": 94,
+			"group": 0,
+			"name": "Krokodil",
+			"shortcut": "E 94",
+			"speed": 90,
+			"weight": 118,
+			"force": 363,
+			"length": 18,
+			"drive": 1,
+			"reliability": 0.8,
+			"cost": 380000,
+			"operationCosts": 120,
+			"equipments": ["AT", "DE","CH","FR"]
```

Comment:
Inwiefern soll die Lok in Frankreich fahren können? Die Unterstützt weder 25 kV noch 1,5 kV…

Reply:
Dann halt noch ein französisches Krokodil dazu